server/plugins/error-pages.js

```diff
@@ -1,26 +1,11 @@
-/*
-* Add an `onPreResponse` listener to return error pages
-*/
+const processErrorResponse = require('./process-error-response')
 
 module.exports = {
   plugin: {
     name: 'error-pages',
     register: (server, options) => {
       server.ext('onPreResponse', (request, h) => {
-        const response = request.response
-
-        if (response.isBoom) {
-          // An error was raised during
-          // processing the request
-          const statusCode = response.output.statusCode
-
-          request.log('error', {
-            statusCode: statusCode,
-            data: response.data,
-            message: response.message
-          })
-        }
-        return h.continue
```

Comment:
The logic has changed slightly from the original boilerplate code - the original code did not call h.continue after the error was logged. I've changed back to the original logic.